README.md

```diff
@@ -208,6 +198,45 @@ export async function getStaticPaths() {
 }
 ```
 
+## Migrate
+
+### From `v0.4`
+
+#### `createPortableTextComponent` is removed
+
+This utility used to wrap `@sanity/block-content-to-react`. It's encouraged to upgrade to `@portabletext/react`.
+
+```sh
+$ npm install @portabletext/react
+// or
+$ yarn add @portabletext/react
+```
+
+```diff
+-import { createPortableTextComponent } from 'next-sanity'
++import { PortableText as PortableTextComponent } from '@portabletext/react'
+
+-export const PortableText = createPortableTextComponent({ serializers: {} })
++export const PortableText = (props) => <PortableTextComponent components={{}} {...props} />
```

Comment:
We should probably make it clearer that `serializers` and `components` are not 100% equivalent - eg custom components probably have to be modified as per the migration guide.

Reply:
How do you feel about the added wording?